snmp.yml

```diff
@@ -1221,3 +1221,377 @@ apcups:
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerRemoteAddr
+          oid: 1.3.6.1.2.1.15.3.1.7
+    - name: bgpPeerHoldTimeConfigured
+      oid: 1.3.6.1.2.1.15.3.1.20
+      indexes:
+        - labelname: bgpPeerIdentifier
+          type: IpAddress
+      lookups:
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerLocalAddr
+          oid: 1.3.6.1.2.1.15.3.1.5
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerRemoteAddr
+          oid: 1.3.6.1.2.1.15.3.1.7
+    - name: bgpPeerKeepAliveConfigured
+      oid: 1.3.6.1.2.1.15.3.1.21
+      indexes:
+        - labelname: bgpPeerIdentifier
+          type: IpAddress
+      lookups:
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerLocalAddr
+          oid: 1.3.6.1.2.1.15.3.1.5
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerRemoteAddr
+          oid: 1.3.6.1.2.1.15.3.1.7
+    - name: bgpPeerMinASOriginationInterval
+      oid: 1.3.6.1.2.1.15.3.1.22
+      indexes:
+        - labelname: bgpPeerIdentifier
+          type: IpAddress
+      lookups:
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerLocalAddr
+          oid: 1.3.6.1.2.1.15.3.1.5
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerRemoteAddr
+          oid: 1.3.6.1.2.1.15.3.1.7
+    - name: bgpPeerMinRouteAdvertisementInterval
+      oid: 1.3.6.1.2.1.15.3.1.23
+      indexes:
+        - labelname: bgpPeerIdentifier
+          type: IpAddress
+      lookups:
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerLocalAddr
+          oid: 1.3.6.1.2.1.15.3.1.5
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerRemoteAddr
+          oid: 1.3.6.1.2.1.15.3.1.7
+    - name: bgpPeerInUpdateElapsedTime
+      oid: 1.3.6.1.2.1.15.3.1.24
+      indexes:
+        - labelname: bgpPeerIdentifier
+          type: IpAddress
+      lookups:
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerLocalAddr
+          oid: 1.3.6.1.2.1.15.3.1.5
+        - labels: [bgpPeerIdentifier]
+          labelname: bgpPeerRemoteAddr
+          oid: 1.3.6.1.2.1.15.3.1.7
+
+# The Cisco BGP4 MIB is proprietary to Cisco routers. Not all routers
+# support all parts of the MIB, even if they are running BGP so your
+# mileage may vary. If possible, upgrade to the latest version of the
+# firmware possible.
+#
+# ftp://ftp.cisco.com/pub/mibs/v2/CISCO-BGP4-MIB.my
+
+cbgp4:
+  version: 2
+  walk:
+    - 1.3.6.1.4.1.9.9.187.1.2
+  metrics:
+    - name: cbgpPeerAcceptedPrefixes
+      oid: 1.3.6.1.4.1.9.9.187.1.2.4.1.1
+      indexes:
+        - labelname: bgpPeerRemoteAddr
+          type: IpAddress
+        - labelname: cbgpPeerAddrFamilyAfi
+          type: Integer32
+        - labelname: cbgpPeerAddrFamilySafi
+          type: Integer32
+      lookups:
+        - labels: [bgpPeerRemoteAddr, cbgpPeerAddrFamilyAfi, cbgpPeerAddrFamilySafi]
+          labelname: cbgpPeerAddrFamilyName
```

Comment:
I think that cbgpPeerAddrFamilyName is strictly in a separate SNMP table, but it's very useful in that it maps (cbgpPeerAddrFamilyAfi, cbgpPeerAddrFamilySafi) to a textual description of the address family, e.g. (1, 1) -> "IPv4 Unicast" or (2, 1) -> "IPv6 Unicast".